libcontainer/container_linux.go

```diff
@@ -290,7 +290,10 @@ func handleFifoResult(result openResult) error {
 	if err := readFromExecFifo(f); err != nil {
 		return err
 	}
-	return os.Remove(f.Name())
+	if err := os.Remove(f.Name()); !errors.Is(err, unix.ENOENT) {
```

Comment:
with `os` functions, it's preferrable to use `errors.Is(os.ErrNotExist)`.

Reply:
done. thanks